openstreetmap-install.sh

```diff
@@ -36,10 +36,16 @@ sudo apt install -y postgresql-server-dev-$pg_version
 # Installing pgxnclient
 sudo apt install -y pgxnclient
 ## pgxn has to call pg_config, add PG pinaries to PATH
-sudo echo 'export PATH=$PATH:/usr/local/pgsql/bin' >> /root/.bashrc
+## sudo echo 'export PATH=$PATH:/usr/local/pgsql/bin' >> /root/.bashrc
 
 # Installing H3
-sudo pgxn install h3
+if [ -d "/usr/lib/postgresql/$pg_version/lib/bitcode/h3/" ] && [ -e "/usr/lib/postgresql/$pg_version/lib/bitcode/h3.index.bc" ] && [ -e "/usr/lib/postgresql/$pg_version/lib/h3.so" ]
```

Comment:
I think checking output of `psql -c "select h3_get_extension_version()"` will be more clear and reliable way.

Reply:
rewrited as a target in installation makefile